src/main/java/org/jenkinsci/plugins/workinghours/WorkingHoursPlugin.java

```diff
@@ -23,23 +23,29 @@
  */
 package org.jenkinsci.plugins.workinghours;
 
+import hudson.ExtensionList;
+import hudson.Plugin;
+import hudson.model.Describable;
+import hudson.model.Descriptor;
+import jenkins.model.GlobalConfiguration;
 import org.jenkinsci.plugins.workinghours.model.TimeRange;
 import org.jenkinsci.plugins.workinghours.model.ExcludedDate;
 import hudson.Extension;
+
 import java.util.Arrays;
 import java.util.Calendar;
 import java.util.Collections;
 import java.util.List;
 import javax.annotation.CheckForNull;
-import jenkins.model.GlobalConfiguration;
+import javax.annotation.Nonnull;
 
 /**
  * Provides configuration options for this plugin.
  *
  * @author jxpearce@godaddy.com
  */
 @Extension(optional = true)
-public class WorkingHoursConfig extends GlobalConfiguration {
+public class WorkingHoursPlugin extends GlobalConfiguration {
```

Comment:
I wonder if we should just delete this class. We can mark the new version as not backward compatible, so people will know they have to reconfigure their settings. The plugin still has vary few users, so it may not be worth much effort figuring out a migration path